kratos/processes/apply_component_table_process.hpp

```diff
@@ -0,0 +1,196 @@
+//    |  /           |
```

Comment:
Please move the code to cpp. Please, change the name to .h (this is a discussion apart, but the 95% of the headers on Kratos are .h, better be consistent)

Reply:
okey that's true